<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>344</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Kalkulator Macierzy v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/matrix.ico</normaloff>:/images/matrix.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="a">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="b">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Macierz A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Macierz B:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="action">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A + B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A * B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B * A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>det(A)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>det(B)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A^-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B^-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A^T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B^T</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="result">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wynik:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderAX">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelAX">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderAY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelAY">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBY">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderBX">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderBY">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelBX">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>kolumn:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>wierszy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>wierszy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>kolumn:</string>
    </property>
   </widget>
   <zorder>a</zorder>
   <zorder>b</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>action</zorder>
   <zorder>result</zorder>
   <zorder>label_7</zorder>
   <zorder>pushButton</zorder>
   <zorder>labelAX</zorder>
   <zorder>horizontalSliderAY</zorder>
   <zorder>labelAY</zorder>
   <zorder>labelBY</zorder>
   <zorder>horizontalSliderBX</zorder>
   <zorder>horizontalSliderBY</zorder>
   <zorder>labelBX</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>horizontalSliderAX</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPomoc">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Pomoc</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionAboutAuthors"/>
    <addaction name="actionTechnicalSupport"/>
   </widget>
   <widget class="QMenu" name="menuPodstawy_teoretyczne">
    <property name="title">
     <string>Podstawy teoretyczne</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionAddition"/>
    <addaction name="actionMultiplication"/>
    <addaction name="actionTransposition"/>
    <addaction name="actionDeterminant"/>
    <addaction name="actionMatrixInverse"/>
   </widget>
   <widget class="QMenu" name="menuTryb_wicze">
    <property name="title">
     <string>Tryb ćwiczeń</string>
    </property>
    <addaction name="actionStartExercise"/>
   </widget>
   <addaction name="menuTryb_wicze"/>
   <addaction name="menuPodstawy_teoretyczne"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTechnicalSupport">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/support</normaloff>:/images/support</iconset>
   </property>
   <property name="text">
    <string>Wsparcie techniczne</string>
   </property>
  </action>
  <action name="actionAddition">
   <property name="text">
    <string>Dodawanie</string>
   </property>
  </action>
  <action name="actionMultiplication">
   <property name="text">
    <string>Mnożenie</string>
   </property>
  </action>
  <action name="actionTransposition">
   <property name="text">
    <string>Transpozycja</string>
   </property>
  </action>
  <action name="actionDeterminant">
   <property name="text">
    <string>Wyznacznik</string>
   </property>
  </action>
  <action name="actionMatrixInverse">
   <property name="text">
    <string>Macierz odwrotna</string>
   </property>
  </action>
  <action name="actionStartExercise">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionAchieve">
   <property name="text">
    <string>Osiągnięcia</string>
   </property>
  </action>
  <action name="actionAboutAuthors">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/help.ico</normaloff>:/images/help.ico</iconset>
   </property>
   <property name="text">
    <string>O autorach</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>a</tabstop>
  <tabstop>b</tabstop>
  <tabstop>action</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>result</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
